etc/db-config.yaml

```diff
@@ -201,6 +201,14 @@
 -   category: Display
     description: Options related to the DOMjudge user interface.
     items:
+        -   name: default_submission_code_mode
```

Comment:
I wonder if this isn't more a preference per user so that having a cookie remembering the last used method is more convenient for the team.

Reply:
I prefer the configuration option. Making this user controlled could still happen secondary.

Reply:
I think we can start by using a configuration option initially, and then use cookies to track once the user starts using it. What do you think?

Reply:
I have already modified it according to the method I mentioned above.